### PR TITLE
[Strategy] Support for Int8 schedules - CUDA/x86

### PR DESCRIPTION
**Pull request by anijain2305**
Recently introduce op strategy currently has some issues with task extraction with AutoTVM. This PR fixes them for x86/CUDA.

@kevinthesun @icemelon9

### CODE REVIEW

**Comment by anijain2305** [comment]
@vinx13 Adding you as well. Because I have padded C dim for GPU using Legalize to use DP4A schedules. Otherwise, we will have to put a check in strategy.

File: topi/python/topi/cuda/conv2d_int8.py [hunk 1]
```diff
@@ -23,10 +24,22 @@
 from .injective import schedule_injective_from_existing
 from .tensor_intrin import dp4a
 from ..nn.pad import pad
+from ..nn.conv2d import unpack_NCHWc_to_nchw
 from ..nn.util import get_pad_tuple
 from ..util import get_const_tuple, traverse_inline
 
 
+def conv2d_nchw_int8(data, kernel, strides, padding, dilation, out_dtype='int32'):
```
Comment (icemelon9) [inline topi/python/topi/cuda/conv2d_int8.py]: add a function doc

**Comment by icemelon9** [comment]
Could you add a few tests for conv2d_nchw_int8 in the topi/tests/python/test_topi_conv2d_int8.py?

otherwise, lgtm

**Comment by vinx13** [comment]
I think padding channels would be helpful, it would be good if we have comparison result (channel padding + int8 template vs direct template)

**Comment by icemelon9** [comment]
Thanks @anijain2305 @kevinthesun @vinx13. This is now merged.

**Pull request merged**